src/api/userApi.ts

```diff
@@ -1,4 +1,4 @@
-import { DataType, ListParams, User } from 'models';
+import {Category, DataType, ListParams, User} from 'models';
```

Comment:
Please exclude the Category import as it will not be utilized for this purpose.

Reply:
I fixed it